tests/python/pants_test/engine/test_rules.py

```diff
@@ -434,18 +438,17 @@ def a_from_suba(suba: SubA) -> A:
     ]
     fullgraph = self.create_full_graph(rules)
 
-    self.assert_equal_with_printing(
-      dedent(
-        f"""\
-        digraph {{
-          // root subject types: SubA
-          // root entries
-            "Select(A) for SubA" [color=blue]
-            "Select(A) for SubA" -> {{"{fmt_rule(a_from_suba)} for SubA"}}
-          // internal entries
-            "{fmt_rule(a_from_suba)} for SubA" -> {{"Param(SubA)"}}
-        }}"""
-      ).strip(),
+    # TODO: we /should/ be able to use dedent() here, but the naive approach causes the string match
```

Comment:
Consider just completely stripping whitespace from these when comparing them, so you don't have to worry about it?

Reply:
Done!!! Thank you for the tip!!!